pipeline/reference.py

```diff
@@ -0,0 +1,189 @@
+'''
+Schema of subject information.
+'''
+import datajoint as dj
+
+schema = dj.schema(dj.config.get('database.prefix', '') + 'gi2017_reference')
+
+
+@schema
+class CorticalLayer(dj.Lookup):
+    definition = """
+    cortical_layer : varchar(8) # layer within cortex
+    """
+    contents = [['N/A'],['1'],['2'],['3'],['4'],['5'],['6'],['2/3'],['3/4'],['4/5'],['5/6']]
+
+
+@schema
+class Hemisphere(dj.Lookup):
+    definition = """
+    hemisphere: varchar(8)
+    """
+    contents = [['left'], ['right']]
+
+
+@schema
+class BrainLocation(dj.Manual): # "dj.Manual" here because, for different session in a dataset, or across different dataset, most likely new applicable brain location will be entered. Unless we have some giant atlas/templates with all brain locations (very unlikely)
+    definition = """ 
+    brain_region: varchar(32)
```

Comment:
It appears that there ought to be a separate lookup that lists out `brain_region` names such as `barrel` or `vm/val`?